include/graphics/twm4nx/twm4nx_config.hxx

```diff
@@ -448,7 +448,7 @@
  */
 
 #ifndef CONFIG_TWM4NX_INPUT_SIGNO
-#  define CONFIG_TWM4NX_INPUT_SIGNO 6
+#  define CONFIG_TWM4NX_INPUT_SIGNO 33
```

Comment:
typo?

Reply:
No, this problem use two signal 5/6 at th same time, we need map to 32/33.